syntax = "proto2";
package oneflow;

import "oneflow/core/job/placement.proto";

message ParallelConfGroup {
  repeated ParallelConf parallel_conf = 1;
}

message ParallelConfGroupList {
  repeated ParallelConfGroup parallel_conf_group = 1;
}
